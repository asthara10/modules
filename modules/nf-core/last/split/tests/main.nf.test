nextflow_process {

    name "Test Process LAST_SPLIT"
    script "../main.nf"
    process "LAST_SPLIT"

    tag "modules"
    tag "modules_nfcore"
    tag "last"
    tag "last/split"

    test("test_last_split") {


        when {
            process {
                """
                
                input[0] = [ [ id:'contigs.genome' ], // meta map
              file(params.test_data['sarscov2']['genome']['contigs_genome_maf_gz'], checkIfExists: true) ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
