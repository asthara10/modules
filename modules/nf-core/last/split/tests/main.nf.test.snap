{
    "versions": {
        "content": [
            [
                
            ]
        ],
        "timestamp": "2023-11-24T13:54:29.617496"
    }
}